src/pages/python.tsx

```diff
@@ -0,0 +1,40 @@
+import '../styles/python.scss';
+import BooleanDropdown from '../components/booleanDropdown';
+
+let i = '';
```

Comment:
i dont really know what you were thinking, but should `i` be a react state?

we can also do it later lol since this is just a test